rfc6962-bis.xml

```diff
@@ -447,12 +447,22 @@ Validation Certificates</xref>.
             When creating a Precertificate, the CA MAY substitute one or more
 labels in each DNS-ID with a corresponding number of
 <spanx style="verb">?</spanx> labels. Every label to the left of a
-<spanx style="verb">?</spanx> label MUST also be a
-<spanx style="verb">?</spanx> label. For example, if a certificate contains a
-DNS-ID of <spanx style="verb">top.secret.example.com</spanx>, then the
-corresponding Precertificate could contain
-<spanx style="verb">?.?.example.com</spanx> instead. Labels in a
-<xref target="RFC6125">CN-ID</xref> MUST remain unredacted.
+<spanx style="verb">?</spanx> label MUST also be redacted. For example, if a
+certificate contains a DNS-ID of
+<spanx style="verb">top.secret.example.com</spanx>, then the corresponding
+Precertificate could contain <spanx style="verb">?.?.example.com</spanx>
+instead, but not <spanx style="verb">top.?.example.com</spanx> instead.
+          </t>
+          <t>
+            Wildcard <spanx style="verb">*</spanx> labels MUST NOT be redacted.
+However, if the complete leftmost label of a DNS-ID is
+<spanx style="verb">*</spanx>, it is considered redacted for the purposes of
+determining if the label to the right may be redacted. For example, if a
+certificate contains a DNS-ID of
+<spanx style="verb">*.top.secret.example.com</spanx>, then the corresponding
+Precertificate could contain
+<spanx style="verb">*.?.?.example.com</spanx> instead, but not
+<spanx style="verb">?.?.?.example.com</spanx> instead.
```

Comment:
A TLS client would see a cert containing *.top.secret.example.com and would discover (from the .2.4.6 certificate extension) that the 3 leftmost labels are redacted in the Precertificate.  The TLS client would know that * is considered redacted, so it would only do two substitutions - "top" for "*" and "secret" for "*" - when it reconstructs the TBSCertificate.
If the TBSCertificate component of the Precertificate contains *.?.?.example.com, then the SCT signature will verify correctly (unless there are other problems).
But if the TBSCertificate component of the Precertificate contains ?.?.?.example.com, the TLS client will consider the SCT signature to be invalid.
(What a TLS client does when it encounters an invalid SCT signature is out of scope for this pull request, I think).